person.rb

```diff
@@ -0,0 +1,15 @@
+# main class person
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age:, name: 'Unknown', parent_permission: true); end
```

Comment:
- [ ] in the initialize method you should create the instance variables as well to make your project work, which can look something like below:
```
def initialize(age:, name: 'Unknown', parent_permission: true);
    @id = Random.rand(1..1000)
    @age = age
    @name = name
    @parent_permission = parent_permission
end
```